docs/config/platformsh.md

```diff
@@ -311,7 +311,7 @@ Then you'd expect to see the following commands and usage:
 
 ```bash
 lando database  Connects to the database relationship
-lando redis  Connects to the database relationship
+lando redis     Connects to the database relationship
 ```
```

Comment:
This line added spaces I'm not sure were meant to be added.

Reply:
@mikemilano yup, intentional, cosmetic change so the command descriptions are in the same column